platform/darwin/src/MGLStyleLayer.mm.ejs

```diff
@@ -121,6 +121,16 @@ namespace mbgl {
 - (void)set<%- camelize(property.name) %>:(NSExpression *)<%- objCName(property) %> {
     MGLAssertStyleLayerIsValid();
 
+<% if (property.tokens) { -%>
```

Comment:
This part of the template is only for layout properties. If paint properties like `background-pattern` can also accept tokens, then this change needs to be repeated further down in the template.

Reply:
`text-field` and `icon-image`, both layout properties, are the only ones that accept tokens, and we don't plan to add any token-accepting properties in the future.